edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -71,37 +81,35 @@ def SetInputVars(self, inputdict):
 
     def FindPath(self, *p):
         """
-
+        Given a path, it will find it relative to the root, the current target file, or the packages path
         Args:
           *p:
 
-        Returns:
+        Returns: None on failure
```

Comment:
on success it is is returning a full absolute path to a file in the local file system

Reply:
updated comment